include/deal.II/base/geometry_info.h

```diff
@@ -2312,6 +2312,19 @@ struct GeometryInfo
                              const bool         face_flip        = false,
                              const bool         face_rotation    = false);
 
+  static unsigned int
+  standard_to_real_line_vertex(const unsigned int line,
+                               const bool         face_orientation = true);
```

Comment:
I'm coming late to this PR, but would you mind adding documentation to these functions?

Reply:
I'll do later!